tests/jest/index.test.ts

```diff
@@ -86,4 +86,11 @@ describe('#validateText()', () => {
 			'The warning level must be one of the following:'
 		);
 	});
+
+	it('Parses out unwanted characters from error messages', async () => {
+		const result = await cssValidator.validateText('.foo { foo: bar; }');
+
+		expect(result.errors.length).toBeGreaterThan(0);
+		expect(result.errors.some((error) => error.message.includes(' : '))).toBe(false);
```

Comment:
Generally, it's better to let Jest do a more meaningful comparison. Otherwise, if this fails, Jest will simply tell us it got `true` instead of `false`. What would be more helpful is if Jest told us what it received and indicated it didn't include the string we were looking for. That can be accomplished by writing the test like:

```suggestion
		for (const error of result.errors) {
			expect(error).not.toMatch(/ : /);
		}
```